.lando.yml

```diff
@@ -154,6 +158,8 @@ services:
         # We instruct node to use the Lando CA certificate for HTTPS connections:
         NODE_EXTRA_CA_CERTS: /lando/certs/LandoCA.crt
     build:
+      # This is needed to be able to run lando npx commands:
+      - "mkdir -p /var/www/.npm-global/lib"
```

Comment:
for the `node`user inside the docker container /var/www is the home folder, the directory needs to be there for npx to work, apparently.

Reply:
```yml
tooling:
  npx:
    service: node
```

doesn't work?

Reply:
Seems to work both ways.

Lets add 
```
dir: /app/next
```
there too.

Reply:
@tormi without this line: 

1. lando destroy
2. lando start
3. 
<img width="606" alt="image" src="https://github.com/user-attachments/assets/b8417701-4ca4-4050-a309-3372a830caf3">

Reply:
Ok, thanks for the explanation.

Reply:
I suggest installing `npm` globally instead. This would be a cleaner solution:

```yml
services:
  node:
    type: node:20
    globals:
      npm: latest
```

Reply:
nice! I tested it and it works, let's switch to that.

Committed in faefbacaa5db0330953ef597cdf95438f39a56bd